protos/sources/vega/commands/v1/commands.proto

```diff
@@ -31,10 +31,10 @@ message OrderSubmission {
   string market_id = 1;
   // Price for the order, the price is an integer, for example `123456` is a correctly
   // formatted price of `1.23456` assuming market configured to 5 decimal places,
-  // required field for limit orders, however it is not required for market orders
+  // required field for limit orders, however it is not required for market orders.
   // This field is an unsigned integer passed as a string and needs to be scaled using the market's decimal places.
   string price = 2;
-  // Size for the order, for example, in a futures market the size equals the number of units, cannot be negative
+  // Size for the order, for example, in a futures market the size equals the number of units, cannot be negative.
```

Comment:
I don't think the size of an order has anything to do with the market's decimal places?